tests/func/test_version.py

```diff
@@ -0,0 +1,20 @@
+from dvc.main import main
+from dvc.command.version import CmdVersion
+from dvc.cli import parse_args
+
+from tests.basic_env import TestDvc
+
+
+class TestVersion(TestDvc):
+    def test_run(self):
+        ret = main(["version"])
+        self.assertEqual(ret, 0)
+
+
+def test_info(dvc, mocker, caplog):
```

Comment:
It doesn't look like you are using `dvc` and `mocker` fixtures, if so, please remove them.